docs/UpgradeGuide.md

```diff
@@ -171,7 +201,78 @@ end
+| `Datadog.tracer.active_root_span`                                                            | Removed                                     | Consider `Datadog::Tracing.active_trace` instead.                   |
+| `Datadog.tracer.active_span.context.sampling_priority = Datadog::Ext::Priority::USER_KEEP`   | `Datadog::Tracing.keep!`                    | See [sampling](#1.0-trace-api-sampling) for details.                |
+| `Datadog.tracer.active_span.context.sampling_priority = Datadog::Ext::Priority::USER_REJECT` | `Datadog::Tracing.reject!`                  | See [sampling](#1.0-trace-api-sampling) for details.                |
+| `Datadog.tracer.active_span.context`                                                         | Removed                                     | Consider `Datadog::Tracing.active_trace` instead.                   |
+| `Datadog.tracer.active_span`                                                                 | `Datadog::Tracing.active_span`              | See [trace state](#1.0-trace-api-trace-state) for details.          |
+| `Datadog.tracer.call_context`                                                                | Removed                                     | See [trace state](#1.0-trace-api-trace-state) for details.          |
+| `Datadog.tracer.configure(options)`                                                          | `Datadog.configure { \|c\| ... }`           | Use configuration API instead.                                      |
+| `Datadog.tracer.provider.context = context`                                                  | `Datadog::Tracing.continue_from!(digest)`   | See [distributed tracing](#1.0-trace-api-distributed) for details.  |
+| `Datadog.tracer.set_tags(tags)`                                                              | `Datadog.configure { \|c\| c.tags = tags }` |                                                                     |
+| `Datadog.tracer.shutdown!`                                                                   | `Datadog::Tracing.shutdown!`                |                                                                     |
+| `Datadog.tracer.start_span`                                                                  | `Datadog::Tracing.trace`                    | See [manual tracing](#1.0-trace-api-manual-tracing) for details.    |
+| `Datadog.tracer.trace`                                                                       | `Datadog::Tracing.trace`                    | See [manual tracing](#1.0-trace-api-manual-tracing) for details.    |
+
+
+Also check out the functions defined within `Datadog::Tracing` in our [public API](https://www.rubydoc.info/gems/ddtrace/) for more details on their usage.
+
+<h3 id="1.0-trace-api-removed-tracer">Removed `Datadog.tracer`</h3>
+
+Many of the functions accessed directly through `Datadog.tracer` have been moved to `Datadog::Tracing` instead.
+
+<h3 id="1.0-trace-api-removed-context">Removed access to `Datadog::Context`</h3>
```

Comment:
This is expected. Markdown does not support custom IDs for headers: it will always use the header's name as an ID. This will inevitably cause conflict if there are subsections bearing the same name.

Reply:
Indeed. My comment was wondering if it made sense to remove the markdown (the backquotes) in this case, since it gets rendered literally.

Reply:
Which I think is fine: helps make the keyword clear, even if it doesn't apply the beautiful styling you would normally expect.